mavros_extras/src/plugins/distance_sensor.cpp

```diff
@@ -222,17 +222,14 @@ class DistanceSensorPlugin : public MavRosPlugin {
 
 		if (sensor->send_tf) {
 			/* variables init */
-			// XXX #319
 			auto rpy = UAS::sensor_orientation_matching(static_cast<MAV_SENSOR_ORIENTATION>(dist_sen.orientation));
-			// how it can work, if rpy in degrees, not radians?
 			auto q = tf::createQuaternionFromRPY(rpy.x(), rpy.y(), rpy.z());
```

Comment:
For example `Eigen::Vector3d v = Eigen::Vector3d(x, y, z) * DEG_TO_RAD;` also does calculations on each call (and are exactly the same as the `from_degrees()` function). So I don't really see your point. You are forcing a problem where it doesn't exist.